lib/build.js

```diff
@@ -0,0 +1,85 @@
+// Additional files to be included in the NPM package
+const includeFiles = [
```

Comment:
when this is merged, could you make an issue that we can add to the read.me that popper.js will need to be added manually to the project

Reply:
TODO